frontend/packages/component-library/src/cms/header/header.module.scss

```diff
@@ -0,0 +1,323 @@
+@use '@code-dot-org/component-library-styles/mixins.scss' as mixins;
+@use '@code-dot-org/component-library-styles/typography.module.scss' as
+  typography;
+@use '@code-dot-org/component-library-styles/variables.scss' as variables;
+
+$desktop-breakpoint: 1268px;
+$tablet-breakpoint: 1023px;
+$mobile-breakpoint: 425px;
+$box-shadow: 0 2px 6px 0 rgb(0 0 0 / 0.08);
+
+// Header common styles
+.headerNavigation {
+  background-color: var(--background-brand-teal-primary);
+  padding: 0.375rem 0.875rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+// Main links styles
+.mainLinksWrapper {
+  display: flex;
+  align-items: center;
+  gap: 0.75rem;
+
+  .homeLink {
+    color: transparent;
+    text-decoration: none;
+
+    @include mixins.transition-all;
+
+    &:hover {
+      opacity: 90%;
+    }
+
+    &:focus-visible {
+      @include mixins.focus-styles;
+      outline-color: var(--neutral-white-alpha-70);
+      border-radius: 1px;
+    }
+
+    & > figure {
+      height: 2.375rem;
+      width: 2.375rem;
+      border-radius: unset;
+    }
+  }
+
+  .mainLinks {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    align-items: center;
+
+    li {
+      margin: 0;
+      padding: 0;
+    }
+
+    a {
+      @include typography.body-three;
+      color: var(--text-neutral-white-fixed);
+      text-decoration: none;
+      padding: 1rem;
+      border-inline: 2px solid transparent;
+
+      @include mixins.transition-all;
+
+      &:hover,
+      &:active {
+        background-color: var(--background-brand-teal-strong);
+      }
+
+      &:focus-visible {
+        border-inline: 2px solid var(--neutral-white-alpha-70);
+        outline: none;
+      }
+    }
+  }
+}
+
+// Button links styles
+.buttonLinks {
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+
+  .signIn.mainMenu,
+  .newProject {
+    background: transparent;
+
+    &:hover {
+      background: var(--background-brand-teal-strong);
+    }
+  }
+
+  .createAccount.mainMenu,
+  .goToDashboard.mainMenu {
+    color: var(--text-brand-teal-primary-fixed);
+
+    &:hover {
+      background: var(--neutral-white-alpha-90);
+    }
+  }
+
+  button,
+  a {
+    &:focus-visible {
+      outline-color: var(--neutral-white-alpha-70);
+    }
+
+    span {
+      font-weight: 500;
+    }
+  }
+}
+
+// Icon button styles
+.buttonLinks .iconButton {
+  padding: 0;
+  background: transparent;
+  min-width: unset;
+  width: 1.75rem;
+  height: 1.75rem;
+  border-radius: 100%;
+
+  & > i {
+    font-size: 1.375rem;
+  }
+
+  &:hover,
+  &:focus {
+    color: var(--neutral-white-alpha-90);
+    background: transparent;
+  }
+
+  &.helpButton {
+    margin-inline: 0.25rem -0.5rem;
+
+    [dir='rtl'] & {
+      transform: scaleX(-1);
```

Comment:
super nit: are we sure we need this? (won't regular change of content order be enough?)